src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/ChooseConnection.java

```diff
@@ -30,17 +32,22 @@
 	private static final long serialVersionUID = 4773092967249064165L;
 
 	private final ToggleButtonIcon bConnect = new ToggleButtonIcon(
-			Arrays.asList("ButtonConnect", "ButtonDisconnect"),
-			Arrays.asList("/images/connect.png", "/images/disconnect.png"),
-			Arrays.asList(Color.GREEN, Color.RED));
```

Comment:
this looks like it should have been one array containing two instances of a class.

Reply:
Done in https://github.com/MarginallyClever/Makelangelo-software/pull/594